code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -316,50 +358,49 @@
 /datum/reagent/consumable/ethanol/beepsky_smash
 	name = "Beepsky Smash"
 	id = "beepskysmash"
-	description = "Deny drinking this and prepare for THE LAW."
+	description = "Drink this and prepare for the LAW."
 	color = "#664300" // rgb: 102, 67, 0
-	boozepwr = 25
+	boozepwr = 90 //THE FIST OF THE LAW IS STRONG AND HARD
 	metabolization_rate = 0.8
 
 /datum/reagent/consumable/ethanol/beepsky_smash/on_mob_life(mob/living/M)
-	M.Stun(2)
-	..()
-	return 1
+	M.Stun(2, 0)
+	return ..()
 
 /datum/reagent/consumable/ethanol/irish_cream
 	name = "Irish Cream"
 	id = "irishcream"
-	description = "Whiskey-imbued cream, what else would you expect from the Irish."
+	description = "Whiskey-imbued cream, what else would you expect from the Irish?"
 	color = "#664300" // rgb: 102, 67, 0
-	boozepwr = 35
+	boozepwr = 70
 
-/datum/reagent/consumable/ethanol/manly_dorf
+/datum/reagent/consumable/ethanol/manly_dorf //THE SPRITE FOR THIS HAS A HANDLE AND THE HANDLE HAD A STRIP OF WHITE PIXELS BETWEEN IT AND THE MUG AND I FINALLY REMOVED IT AND OH GOD I'M SO HAPPY
```

Comment:
The current sprite for Manly Dorf is a mug with a handle. Between the handle and mug is a block of white pixels that has irked me endlessly. In this PR I removed the block of white pixels to make it look more like an actual cup.